MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -186,8 +187,13 @@ object VisionManager : BaseVisionManager {
         sensorsManager = SensorsManager.Impl(application)
         locationEngine = LocationEngine.Impl(application)
 
-        val videoRecorder = SurfaceVideoRecorder.MediaCodecPersistentSurfaceImpl(application)
-        (videoSource as? Camera2VideoSourceImpl)?.setVideoRecorder(videoRecorder)
+        val videoRecorder = if (videoSource is Camera2VideoSourceImpl) {
```

Comment:
Lets split this to separate commit? It's really bugfix and should be stated distinctively of the rest examples code changes.

Reply:
done